added top margin

rosaliechan opened the pull request:
Fix for https://github.com/NUKnightLab/TimelineJS3/issues/526 @scott2b @JoeGermuska

zachwise commented:
I think there was a reason why those were commented out. We need to make sure we do a lot of testing on this.

rosaliechan commented:
I think the reason why they were commented out was that making it vertically aligned doesn't do anything -- the media just stays at the top anyway. That's why I added a 10% top margin. After testing though, I realized this margin would negatively affect vertical images in terms of scrolling. I just fixed it so that TL.Media.less is changed back to what it was before, and the 10% top margin only affects block quotes, since block quotes are more likely to look off if they are at the top. @zachwise

scott2b commented:
closing for now
